File: dlang/psi-impl/src/main/java/io/github/intellij/dlanguage/stubs/DlangFileStub.java
package io.github.intellij.dlanguage.stubs;

import com.intellij.psi.stubs.PsiFileStubImpl;
import io.github.intellij.dlanguage.psi.DlangPsiFileImpl;

/**
 * Basic implementation of a stub for a D file so we can index its contents.
 */
public class DlangFileStub extends PsiFileStubImpl<DlangPsiFileImpl> {
    public DlangFileStub(final DlangPsiFileImpl file) {
        super(file);
    }
}

File: dlang/psi-impl/src/main/kotlin/io/github/intellij/dlanguage/stubs/DlangFileStub.kt
package io.github.intellij.dlanguage.stubs

import com.intellij.psi.stubs.PsiFileStubImpl
import io.github.intellij.dlanguage.psi.DlangPsiFileImpl

/**
 * Basic implementation of a stub for a D file so we can index its contents.
 */
class DlangFileStub(file: DlangPsiFileImpl?) : PsiFileStubImpl<DlangPsiFileImpl>(file)

